pkg/curatedpackages/bundlesource.go

```diff
@@ -0,0 +1,31 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"strings"
+)
+
+type BundleSource string
+
+const (
+	Cluster  = "cluster"
+	Registry = "registry"
+)
+
+func (b BundleSource) String() string {
+	return string(b)
+}
+
+func (b *BundleSource) Set(s string) error {
+	switch strings.ToLower(s) {
+	case Cluster, Registry:
+		*b = BundleSource(s)
+	default:
+		return fmt.Errorf("unknown source: %q", s)
+	}
+	return nil
+}
+
+func (b BundleSource) Type() string {
```

Comment:
What is this for?

Reply:
This is to validate the bundlesource i.e. the user is only able to pass in cluster and registry for source flag.